.claude/scripts/mcp_common.sh

```
@@ -0,0 +1,2130 @@
+        echo -e "${GREEN}  ✅ Successfully added $name to ${MCP_SCOPE} scope${NC}"
+        log_with_timestamp "Successfully added MCP server: $name (scope: ${MCP_SCOPE})"
+
+        # If dual-scope installation is enabled and we just installed to local, also install to user scope
+        if [ "$MCP_INSTALL_DUAL_SCOPE" = true ] && [ "$MCP_SCOPE" = "local" ]; then
+            echo -e "${BLUE}  🔄 Also installing $name to user scope for system-wide availability...${NC}"
+            local user_add_output
+            local user_add_exit_code
+
+            # Build user scope command (same as local but with --scope user)
+            if [ "$name" = "grok-mcp" ]; then
+                local user_add_cmd=(${MCP_CLI_BIN} mcp add --scope user "${cli_args[@]}" "${grok_env_flags[@]}" "$name" "$NODE_PATH" "$grok_path" "${cmd_args[@]}")
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)